docs/test-suites.md

```diff
@@ -0,0 +1,30 @@
+# Testing
+
+During the development if an issue occurs, we try to program the type system
+to reject codes which could lead to similar issues in the future. The second
+defense line is to write quick check properties that checks against the
+assumptions we have at the moment.
+
+As the test suite grows it is necessary to make a separation between the
+tests which we run during the development cycle and those tests that should
+be run on the continuous integration. We call the first one as **smoke
+suite**, the second one we can ommit the 'smoke' preffix, or it can be called the **large suite**.
```

Comment:
"we call it smoke suite" (no "as"); "omit"; "prefix"; can you break the line one more time?